fetcher.go

```diff
@@ -186,3 +200,5 @@ var DefaultPrototypeChooser = dagpb.AddSupportToChooser(func(lnk ipld.Link, lnkC
 	}
 	return basicnode.Prototype.Any, nil
 })
+
+var DefaultReifier = unixfsnode.Reify
```

Comment:
Having this as a default in this library seems maybe kinda questionable to me.  I don't know exactly where we'd put this library on a graph/layercake/whatever-drawing-format of dependencies and what semantic level we expect of them.  But in general I'd probably try to push this as high as possible.

Reply:
yea no it doesn't have to go there. I'm mixed about it being there. I think it ties us too much to the IPFS case. Think I'll take it out.